src/modules/core/components/ArgumentDisplay/ArgumentDisplay.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+
+import HookedUserAvatar from '~users/HookedUserAvatar';
+import TimeRelative from '~core/TimeRelative';
+import Tag from '~core/Tag';
+import { useUser } from '~data/index';
+import { getMainClasses } from '~utils/css';
+
+import styles from './ArgumentDisplay.css';
+
+interface Appearance {
+  theme?: 'motion' | 'objection';
+}
+
+interface Argument {
+  author: string;
+  comment: string;
+  timestamp: number;
+}
+
+interface Props {
+  appearance?: Appearance;
+  argument: Argument;
+}
+
+const displayName = 'ArgumentDisplay';
+
+const ArgumentDisplay = ({
+  appearance = { theme: 'motion' },
+  argument: { author, timestamp, comment },
+}: Props) => {
+  const user = useUser(author);
+  const tagTheme = appearance.theme === 'motion' ? 'primary' : 'pink';
+  const tagText = appearance.theme === 'motion' ? 'Motion' : 'Objection';
```

Comment:
This is a bit too trivial (think of cases where we could have more themes)
We could for example create some MAP object which would return correct arguments for theme

Reply:
Decided to make it simple because we would only have 2 themes for now. But yeah I could make a more dynamic way of choosing the themes for the Tag

Reply:
I agree with Alicja. I would suggest to have the same theme names as in the `Tag` and just pass it directly.
Also I would suggest to pass a prop with the tag text (by the way it should be done with react intl I think).
This way you decouple appearance and contents and it will give you more flexibility for potential future components.

Reply:
I like that idea! I'll implement it